filebeat/docs/multiple-prospectors.asciidoc

```diff
@@ -18,8 +23,13 @@ filebeat:
     -
       paths:
         - "/var/log/apache2/*"
+      fields:
+        json: true
```

Comment:
change 'json' to 'apache'. Setting 'json: true' is kinda unrealistic for apache logs